packages/integration-sdk-testing/src/jobState.ts

```diff
@@ -100,7 +102,7 @@ export function createMockJobState({
     addEntities,
 
     addRelationship: async (relationship: Relationship) => {
-      addRelationships([relationship]);
+      await addRelationships([relationship]);
```

Comment:
This is the `addRelationships` floating promise I mentioned in the PR description